ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1202,3 +1202,14 @@ literals:
 - thermal_energy_accumulator
 - ground_isolation_command
 - ground_temperature_sensor
+
+## New fields for Freezer (Walk in Cold room)
+- defrost_run_status
+	- ON
+	- OFF
+- defrost_temperature_sensor
+- alert_reset_command
+	- NORMAL
```

Comment:
Syntax is incorrect. Please fix.

Reply:
- alert_reset_command has been deleted in this pull request since it was discussed and  recommended in the issue request #502